src/app/app/create/page.jsx

```diff
@@ -1,52 +1,18 @@
-"use client"
-import { Button } from "@/components/ui/button"
-import { Input } from "@/components/ui/input"
-import { CircleCheck, CircleX, Loader2 } from "lucide-react"
-import { useState } from "react"
-import isURL from "validator/lib/isURL"
+import { authOptions } from "@/app/api/auth/[...nextauth]/route";
+import CreateForm from "@/components/createForm";
+import { getServerSession } from "next-auth";
 
-export default function CreatePage() {
-
-    // TODO: replace navbar "Create" with "App" which is a dropdown that includes: Create, Analytics, Dashboard, etc...
-
-    const [inputVal, setInputVal] = useState("")
-    const [isValid, setIsValid] = useState(false)
-    const [isSubmitting, setIsSubmitting] = useState(false)
-
-    const handleInputChange = (e) => {
-        setIsValid(isURL(e.target.value, {
-            require_protocol: true
-        }))
-        setInputVal(e.target.value.trim())
-        console.log('input changed')
-    }
-
-    const handleClick = () => {
-        // TODO: replace once API implemented
-        console.log('button click')
-        setIsSubmitting(true)
-        setTimeout(() => {
-            setIsSubmitting(false)
-        }, 3000);
-        // window.location.replace(inputVal)
-    }
+export default async function CreatePage() {
+    const session = await getServerSession(authOptions);
 
+    // TODO: possibly put create 'form' in its own component so that the outer page can be an rsc and access session obj faster
```

Comment:
nit: remove comment, this has already been done although not really necessary as outer page doesn't hold a ton of data that would benefit from SSR/RSC. Still good to separate create form into its own component